src/account/ResetPassword.module.css

```
@@ -0,0 +1,130 @@
+/* Extra small devices (phones, 600px and down) */
+@media only screen and (max-width: 600px) {
+    .form-div {
+        height: 400px;
+        width: 300px;
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        flex-direction: column;
+        border: 5px solid #afd3d2;
+        border-radius: 2.5rem;
+        box-shadow:
+            0 4px 8px 0 rgba(0, 0, 0, 0.2),
+            0 6px 20px 0;
+    }
+
+    .reset-password-btn {
+        background-color: #71aca9;
+        color: white;
+        border: 2px solid #71aca9;
+        border-radius: 2.5rem;
+        width: 80%;
+        height: 2.5rem;
+        font-size: 1.25rem;
+        font-weight: bold;
+        margin: 20px;
+    }
+}
+
+/* Small devices (portrait tablets and large phones, 600px and up) */
+@media only screen and (min-width: 600px) {
+    .form-div {
+        height: 400px;
+        width: 70vw;
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        flex-direction: column;
+        border: 5px solid #afd3d2;
+        border-radius: 2.5rem;
+        box-shadow:
+            0 4px 8px 0 rgba(0, 0, 0, 0.2),
+            0 6px 20px 0;
+    }
+
+    .reset-password-btn {
+        background-color: #71aca9;
+        color: white;
+        border: 2px solid #71aca9;
+        border-radius: 2.5rem;
+        width: 80%;
+        height: 2.5rem;
+        font-size: 1.25rem;
+        font-weight: bold;
+        margin: 20px;
+    }
+}
+
+/* Medium devices (landscape tablets, 768px and up) */
+@media only screen and (min-width: 768px) {
+    .form-div {
+        height: 450px;
+        width: 50vw;
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        flex-direction: column;
+        border: 5px solid #afd3d2;
+        border-radius: 2.5rem;
+        box-shadow:
+            0 4px 8px 0 rgba(0, 0, 0, 0.2),
+            0 6px 20px 0 rgba(0, 0, 0, 0.19);
+    }
+}
+
+.container-fluid {
+    min-height: calc(100vh - 4rem);
```

```suggestion
    margin: 3rem 0;
```
đổi lại do t chỉnh navbar ko có height nữa